docs/source/examples/Using Interact.ipynb

```diff
@@ -898,6 +898,96 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "### `interactive_output`\n",
+    "\n",
+    "The `interactive_output` function provides maximum flexibility at the cost of a bit more code.\n",
```

Comment:
I think the word 'flexibility' is a bit vague. How about:

`interactive_output` provides additional flexibility: you can control how the UI elements are laid out.